src/pages/home/report/ContextMenu/ContextMenuUtils/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+
+/**
+ * We should show popover description only for mWeb
+ *
+ * @param {Boolean} isSmallScreenWidth
+ * @returns {Boolean}
+ */
+function shouldShowDescription(isSmallScreenWidth) {
+    return isSmallScreenWidth;
+}
+
+export {
+    shouldShowDescription,
+};
```

Comment:
```suggestion
function getPopoverDescription(selection, isSmallScreenWidth) {
    return isSmallScreenWidth ? selection : '';
}

export {
    getPopoverDescription,
};
```

Reply:
Change the method name to be more descriptive and perform slightly different logic.